### PR TITLE
Document extensionless file packing

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2054

### CODE REVIEW

File: docs/reference/msbuild-targets.md [hunk 1]
```diff
@@ -251,6 +251,25 @@ When packing a license file, you need to use PackageLicenseFile property to spec
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="LICENSE" Pack="true" Visible="false" PackagePath=""/>
+  </ItemGroup>  
+```
+
+[File without an extension sample](https://github.com/NuGet/Samples/blob/master/PackageLicenseFileExtensionlessExample/).
+
+
```
Comment (loic-sharma) [inline docs/reference/msbuild-targets.md:272]: Super nit: there's an extra line here

```suggestion

```

File: docs/reference/msbuild-targets.md [hunk 2]
```diff
@@ -251,6 +251,25 @@ When packing a license file, you need to use PackageLicenseFile property to spec
 
 [License file sample](https://github.com/NuGet/Samples/tree/master/PackageLicenseFileExample).
 
+### Packing a file without an extension
+
+In some scenarios, like when packing a license file, you might want to include a file without an extension.
+For historical reasons, NuGet & MSBuild treat paths without an extension as directories
```
Comment (loic-sharma) [inline docs/reference/msbuild-targets.md:257]: Woah this is some wizardy. Should we specifically call out what `Visible` is and what it does here? Is there a link to docs we could add?

Comment (loic-sharma) [inline docs/reference/msbuild-targets.md:257]: Don't forget a period! :)
```suggestion
For historical reasons, NuGet & MSBuild treat paths without an extension as directories.
```

**Review by loic-sharma — APPROVED** [review]

File: docs/reference/msbuild-targets.md (hunk 2 above)
Comment (nkolev92) [reply docs/reference/msbuild-targets.md]: I'll just remove it :D 
I don't want to distract from the point of this change.

**Pull request merged**